FEXCore/Source/Interface/Core/ArchHelpers/Arm64Emitter.cpp

```diff
@@ -57,6 +57,12 @@ namespace x64 {
     ARMEmitter::Reg::r24, ARMEmitter::Reg::r25, ARMEmitter::Reg::r30, ARMEmitter::Reg::r18,
   };
 
+  // p6 and p7 registers are used as temporaries no not added here for RA
+  // See PREF_TMP_16B and PREF_TMP_32B
+  // Also p8-p15 cannot be used can only encode p0-p7, so we're left with p0-p5.
+  constexpr std::array<ARMEmitter::PRegister, 6> PR = {ARMEmitter::PReg::p0, ARMEmitter::PReg::p1, ARMEmitter::PReg::p2,
```

Comment:
A thing of note, p0 and p1 are used as temporaries in the JIT for various things. So this actually leaves p2-p5 for allocation. Very few, but should be fine for our use cases still.

Reply:
Right - maybe this could be cleaned up and use RA? What do you think?

Reply:
They are internal implementation details, I think RAing the temps would be more annoying than it is worth.
Similar to the GPRs and FPRs having temporaries in the backend, I think we just need to label them `PTMP1` and `PTMP2`, to ensure the backend doesn't overstep and use RA allocated ones.

Reply:
Ok - make sense.